azure-pipelines.yml

```diff
@@ -12,16 +12,20 @@ variables:
   - group: sonarqube
   - group: sign
   - group: ossrh
+  - group: changelog
 
 resources:
   repositories:
     - repository: templates
       type: github
       name: bakdata/bakdata-project-templates
+      ref: tmp/upload-snapshot
```

Comment:
TODO

Reply:
I always remember to late 😬 Updated already! :D

Reply:
That's what reviews are for :D